src/splinterdb.c

```diff
@@ -686,3 +686,11 @@ splinterdb_stats_reset(splinterdb *kvs)
 {
    trunk_reset_stats(kvs->spl);
 }
+
+void
```

Comment:
A minor single-line comment will help w/ readability. (I understand that it's a minimalistic function.)

Reply:
Thanks, done in latest.